### PR TITLE
nmcli-monitor: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
For #6167

**Review by bl-ue — APPROVED** [review]

File: pages/linux/nmcli-monitor.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor NetworkManager showing changes with connection status and similar.
+> More information: <https://developer.gnome.org/NetworkManager/stable/nmcli.html>.
+
+- Monitor NetworkManager changes:
+
+`nmcli monitor`
```
Comment (mfrw) [inline pages/linux/nmcli-monitor.md:8]:
```suggestion
`nmcli monitor`

- List connection profiles:

`nmcli connection show`
```

**Review by mfrw — APPROVED** [review]
Added one more example :)

File: pages/linux/nmcli-monitor.md (hunk 1 above)
Comment (bl-ue) [reply pages/linux/nmcli-monitor.md:8]: This page is about `nmcli monitor` though, and the new command is `nmcli connection` 🤔

Comment (mfrw) [reply pages/linux/nmcli-monitor.md:8]: I became too enthusiatic when i saw `nmcli` .. i use this `connection` subcommand often... :( my bad

Comment (CleanMachine1) [reply pages/linux/nmcli-monitor.md:8]: We'll get to it, just remember to bring it up if I forget when revamping the nm-connect page

File: pages/linux/nmcli-monitor.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor NetworkManager showing changes with connection status and similar.
```
Comment (sbrl) [inline pages/linux/nmcli-monitor.md:3]: Perhaps this will read better?
```suggestion
> Monitor changes to the NetworkManager connection status.
```

Comment (CleanMachine1) [reply pages/linux/nmcli-monitor.md:3]: See what others think, I don't like the `the` and without it, it doesn't work either

Comment (CleanMachine1) [reply pages/linux/nmcli-monitor.md:3]: @SethFalco @navarroaxel

Comment (navarroaxel) [reply pages/linux/nmcli-monitor.md:3]: I like sbrl suggestion

File: pages/linux/nmcli-monitor.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor changes to the NetworkManager connection status.
+> More information: <https://developer.gnome.org/NetworkManager/stable/nmcli.html>.
+
+- Monitor NetworkManager changes:
```
Comment (marchersimon) [inline pages/linux/nmcli-monitor.md:6]:
```suggestion
- Start monitoring NetworkManager changes:
```
What do you think?

Comment (CleanMachine1) [reply pages/linux/nmcli-monitor.md:6]: Doesn't really add anything IMO

Comment (marchersimon) [reply pages/linux/nmcli-monitor.md:6]: I think it just fits better, but if you prefer the original one I'm okay with that.

Comment (CleanMachine1) [reply pages/linux/nmcli-monitor.md:6]: Start monitoring is only used once in every page, so it seems outnumbered and out used.

Comment (sbrl) [reply pages/linux/nmcli-monitor.md:6]: I like @marchersimon's suggestion better in this instance.

**Review by marchersimon — APPROVED** [review]
👍🏽

**Pull request merged**